src/Tribe/Assets.php

```diff
@@ -0,0 +1,116 @@
+<?php
+class Tribe__Tickets__Assets {
+	/**
+	 * Enqueue scripts for front end
+	 *
+	 * @since TBD
+	 */
+	public function enqueue_scripts() {
+		tribe_assets(
+			Tribe__Tickets__Main::instance(),
+			array(
+				array( 'event-tickets-tickets-css', 'tickets.css', array( 'dashicons' ) ),
+				array( 'event-tickets-tickets-rsvp-css', 'rsvp.css', array() ),
+				array( 'event-tickets-tickets-rsvp-js', 'rsvp.js', array( 'jquery', 'jquery-ui-datepicker' ) ),
+				array( 'event-tickets-attendees-list-js', 'attendees-list.js', array( 'jquery' ) ),
+			),
+			'enqueue_scripts'
+		);
+	}
+
+	/**
+	 * Enqueue scripts for admin views
+	 *
+	 * @since TBD
+	 */
+	public function admin_enqueue_scripts() {
+		global $post;
+
+		/**
+		 * Filter the array of module names.
+		 *
+		 * @since TBD
+		 *
+		 * @param array the array of modules
+		 *
+		 * See event-tickets/src/Tribe/Tickets.php->modules()
+		 */
+		$modules = apply_filters( 'tribe_events_tickets_modules', null );
+
+		// For the metabox
+		if ( ! empty( $post ) &&
```

Comment:
Let's bump the first part of the conditional to the next line so we're following [this standard](http://moderntribe.github.io/products-engineering/php/#php-control-multi) - we should also move hte `&&` operator to the start of the line that follows - ie:

```php
if (
	$first
	&& $second
	&& $third
) {
	// Body
}
```